sbin/jtreg.sh

```diff
@@ -105,32 +106,32 @@ packageTestResultsWithJCovReports()
   echo "Package test output into archives..." 
   pwd
 
-  cd $WORKING_DIR/$OPENJDK_REPO_NAME/build/$BUILD_FULL_NAME/
+  cd "$WORKING_DIR/$OPENJDK_REPO_NAME/build/$BUILD_FULL_NAME/" || exit
  
-  artifact=${JOB_NAME}-testoutput-with-jcov-reports
+  artifact="${JOB_NAME}-testoutput-with-jcov-reports"
   echo "Tarring and zipping the 'testoutput' folder into artefact: $artifact.tar.gz" 
-  tar -cvzf $WORKING_DIR/$artifact.tar.gz   testoutput/
+  tar -cvzf "$WORKING_DIR/$artifact.tar.gz"   testoutput/
```

Comment:
`v` will enlist all the files being tarred, do we want to show this to the end-user or record it in the logs - in that case its fine